### ISSUE TITLE
Training accuracy could not reach the accuracy of the paper, the difference of 7 percentage points

### ISSUE THREAD

**Issue by jlhou** [megvii-model/ShuffleNet-Series#19]
Thank you so much for such an outstanding job!But my ShuffleNet V1's accuracy on the ImageNet data set is  seven percentage points off your results.The parameters I used are basically the same as yours!Only batchsize is 384, step is 7.8*e5, which is due to the limitation of my equipment.How can I improve the accuracy of the network?

**Comment by nmaac** [comment]
Hi @jlhou , to achieve the same result, you should tune the learning rate if you change the batchsize.

**Comment by jlhou** [comment]
How do I adjust the learning rate？ Whether to scale down the initial learning rate？

**Comment by nmaac** [comment]
Try 0.1875

> How do I adjust the learning rate？ Whether to scale down the initial learning rate？
